pkg/utils/host/host.go

```diff
@@ -46,15 +46,6 @@ func init() {
 }
 
 type Config struct {
-	// AutoMountFilesystems will automatically mount bpffs, debugfs and
```

Comment:
Since the Config struct is now empty, should it be removed from the `func Init()` prototype?